### PR TITLE
Mostly cleanup/structuring change

### PR DESCRIPTION
**Pull request by j10a1n15**

### CODE REVIEW

File: src/commands/mods.js [hunk 1]
```diff
@@ -169,6 +169,9 @@ async function handleAddCommand(client, interaction) {
   }
   modVersions[version] = hash
 
+  mod.versions = modVersions
+  mod.betaVersions = modVersions
```
Comment (Su386yt) [inline src/commands/mods.js:173]: This normal?

Comment (j10a1n15) [reply src/commands/mods.js:173]: its like that in update so yeah ig

Comment (j10a1n15) [reply src/commands/mods.js:173]: So what do I do

Comment (Su386yt) [reply src/commands/mods.js:173]: Just leave it

**Pull request merged**